src/aspire/source/__init__.py

```diff
@@ -543,10 +543,10 @@ def save(
             save metadata of image_source
         :param batch_size: Batch size of images to query.
         :param save_mode: Whether to save all images in a single or multiple files in batch size.
-        :param overwrite: Option to overwrite the output mrcs files.
+        :param overwrite: Option to overwrite the output MRCS files.
         """
         logger.info("save metadata into STAR file")
-        self.save_metadata(
+        mrcs_filenames = self.save_metadata(
```

Comment:
Since this is not just filenames, but indices, it would be good to name it something like `image_indices` or `filename_indices` so that it's clear it's not just filenames.

Reply:
Changed to filename_indices in commit ea7df79.